### DISCUSSION TITLE
Live preview

### DISCUSSION THREAD

**Discussion by gigli69** [BobanSpasic/PSS-Revive#11] [Q&A]
"Live preview" is working: when modifying a parameter it is directly sent to the PSS.
But when "Live preview" is not checked, how to send the sysex to the PSS, the icon "Send Bank over MIDI" seems not to be working => there is no sound modification.

Thanks

**Comment by BobanSpasic** [comment]
Hi.
After the bank is sent, you need to change the program/bank once, so that the CPU in the synth reads the data from the memory. Without that, it still plays the voice with the data from the buffer state before the bank is sent.
